examples/gno.land/p/demo/web/generators.gno

```diff
@@ -0,0 +1,99 @@
+package web
+
+import (
+	"std"
+	"strings"
+	"regexp"
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/demo/avl"
+)
+
+func NewRouter() *Router{
+	return &Router{}
+}
+
+func NewRoute(pattern_ string, action_ RouteAction) *Route {
+	pattern_ = attify(pattern_)	
+	_pathParts := trimSplit(pattern_)
```

Comment:
curious, what's the reason for adding underscores before and after variables?

Reply:
It's a pattern I picked up from following along with Chainlink's standards. They go a step further and global vars that effect state get prefixed with `s_` but I never fully adopted that one. I've just found it makes it easier to tell where vars are coming from (private/param/global) at a glance while avoiding shadowing so it stuck.

Reply:
looks a little bit weird IMO; but let's see later how it goes